### DISCUSSION TITLE
[Money requierement] for WARPS

### DISCUSSION THREAD

**Discussion by Flashback083** [Zrips/CMI#3172] [Feature Requests]
**Description of feature/improvement request:**
First, sorry if the plugin already have it but i can't find where :/ 
The idea is to have a a config that allow warp to cost money when use, and prevent player to use warp if he doesnt have enough money
---
### Detailed explanation of what it should do, when and in what situation: ###
In the gui its a bit difficult to set money maybe so maybe : 
commad like /cmi warpcost <warpname> <cost>
or with the gui add a Cheque or an item that can read the value of item to see how many €
or wi the gui 10 new buttons : (1-4 for remove X € of the price, 6-9 for add X € of the price and the 5 for see the price)
### Basic explanation as to why this feature is needed and/or useful to have: ###
I have some area that players can tp through warp and i wanna this area cost money directly through your plugin 
Thanks a lot for your work !

**Comment by TheBadshah** [comment]
bump
